### PR TITLE
Added phone number field to user

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

**Comment by sandlerben** [comment]
r @thomaslee1911 @kongsally

File: app/account/forms.py [hunk 1]
```diff
@@ -4,10 +4,10 @@
     StringField,
     PasswordField,
     BooleanField,
-    SubmitField
+    SubmitField,
```
Comment (thomaslee1911) [inline app/account/forms.py:6]: Do we need this comma?

File: app/account/forms.py [hunk 2]
```diff
@@ -37,6 +37,12 @@ class RegistrationForm(Form):
         Length(1, 64),
         Email()
     ])
+    phone_number = TelField('Phone Number', validators=[
+        Length(0, 15),
```
Comment (thomaslee1911) [inline app/account/forms.py]: Should the minimum length be 10?

File: app/account/forms.py [hunk 3]
```diff
@@ -37,6 +37,12 @@ class RegistrationForm(Form):
...
+        Regexp(r'^[0-9]*$',
```
Comment (thomaslee1911) [inline app/account/forms.py]: We discussed this a little during meeting, I think we said we'd consider allowing symbols but removing them later?

File: app/account/forms.py (hunk 1 above)
Comment (sandlerben) [inline app/account/forms.py:6]: Technically no, but it's good style to leave an extra comma in situations like this

For example, Go won't even compile if you leave out the dangling comma!

File: app/account/forms.py (hunk 2 above)
Comment (sandlerben) [inline app/account/forms.py]: Apparently for international numbers, 15 is the most it can be.

File: app/account/forms.py (hunk 3 above)
Comment (sandlerben) [inline app/account/forms.py]: Yes, going to do that!

**Comment by sandlerben** [comment]
Now all phone number formats are accepted. The non-digit characters are stripped out.

Look good now @kongsally @thomaslee1911 ?

**Comment by sandlerben** [comment]
Ok, just waiting for a :+1: from @thomaslee1911

**Pull request merged**